labs/Fabric-EVM-Library.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: default
+title: Fabric-EVM-Library Proposal Document
+parent: Fabric-EVM-Library
+---
+# Lab Name
+Fabric-EVM-Library
+
+# Short Description
+
+Fabric-EVM-Library is a complete virtual Blockchain node and Smart Contract execution engine. Fabric-EVM-Library is a distributed product that executes Smart Contracts and application code. Fabric-EVM-Library runs Ethereum Virtual Machine (EVM) and Web Assembly (WASM) Smart Contracts. The Fabric-EVM-Library will bring the option of the EVM to the Hyperledger Fabric developer and user community.
+
+# Scope of Lab
+This lab's scope is to provide the Hyperledger Fabric community with an Ethereum Virtual Machine (EVM) Smart Contract and application solution. The EVM is quite popular for developing Blockchain/ Distributed Ledger Technology (DLT) applications. In addition, the EVM is a market-leading Smart Contract engine that not only runs on the Ethereum network but also runs on many other well-known Blockchain/DLT networks. As a market leader, the EVM has the advantage of its own community supporting numerous modern development tools and standards to develop applications for the most current Blockchain/DLT use cases. This lab intends to allow the Hyperledger Fabric community to use the EVM to develop Smart Contracts and distributed applications (DApps) on the Hyperledger Fabric product. The Fabric-EVM-Library will deliver all the EVM benefits to the Hyperledger Fabric community.
+
+# Initial Committers
+
+ https://github.com/theblockchainacademy/jim.sullivan
+
+# Sponsor
+
+Bobbi Muscara (bobbi@ledgeracademy.com), Technical Steering Committee (TSC) member. <br>
+
+# Pre-existing repository
+Listed below are the product repositories that are all components of this proposal.
+
+<ul> <li>https://github.com/theblockchainacademy/burrow</li>
```

Comment:
Curious : What are we doing for Burrow here ? The Lab description doesn't call it out and I believe Burrow has been marked as EOL in May 2022.
Could you please elaborate.